packages/text-input/text-input.scss

```diff
@@ -7,7 +7,7 @@
 $_text-input-height: jkl.rem(48px);
 $_text-input-height--mobile: jkl.rem(44px);
 $_text-input-height--compact: jkl.rem(32px);
-$_text-input-padding: jkl.$spacing-xs jkl.$spacing-xs jkl.$spacing-xs jkl.$spacing-s;
+$_text-input-padding: jkl.rem(6px) jkl.$spacing-xs jkl.rem(6px) jkl.$spacing-s;
```

Comment:
Dette funker jo for så vidt fint, men jeg er ikke superhappy for å ha "magiske tall" her som ikke stemmer med skissene. Så vidt jeg kan se etter å ha testet en del kan man beholde riktig padding her (`jkl.$spacing-xs`) dersom man fjerner `padding-bottom` som er satt på linjene 304 og 311.

Så vidt jeg kan se ødelegger det ikke TextArea, og det er fortsatt ingen clipping av "Å" i Chrome, dersom man gjør det på den måten :)

Reply:
Bare så vi snakker om samme padding-bottom: er det linje 306 og 310 du mener @piofinn?